packages/js-drive/lib/core/fetchTransactionFactory.js

```diff
@@ -0,0 +1,33 @@
+const { Transaction } = require('@dashevo/dashcore-lib');
+
+/**
+ * @param {RpcClient} coreRpcClient
+ * @returns {fetchTransaction}
+ */
+function fetchTransactionFactory(coreRpcClient) {
+  /**
+   * @typedef {fetchTransaction}
+   * @param {string} id
+   * @returns {Transaction}
+   */
+  async function fetchTransaction(id) {
+    let rawTransaction;
+
+    try {
+      ({ result: rawTransaction } = await coreRpcClient.getRawTransaction(id, 1));
+    } catch (e) {
+      // Invalid address or key error
```

Comment:
I think if it times out you should retry. Not essential for this version but still something todo.

Reply:
That's a good idea. We have something about connection reliability with Core in our backlog.